prometheus-rules/templates/prometheusrules.k8s.yaml

```diff
@@ -24,4 +24,46 @@ spec:
           annotations:
             summary: 'Job {{ "{{" }} $labels.namespace }}/{{ "{{" }} $labels.job_name }} failed to complete.'
             description: "Job"
+
+    - name: kubernetes-storage
+      rules:
+      - alert: KubePersistentVolumeUsageCritical
+        annotations:
+          description: 'The PersistentVolume claimed by {{ "{{" }} $labels.persistentvolumeclaim }} in Namespace {{ "{{" }} $labels.namespace }} is only {{ "{{" }} printf "%0.2f" $value }}% free.'
```

Comment:
Is `{{ "{{" }} ... }}` an expected syntax? Looks like a recurring type

Reply:
Yes, it is expected. Helm chart interprets `{{` as it's own variable.